SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -22,14 +22,16 @@
   <!-- **************************************************************************** -->
   <!-- Using tasks -->
   <!-- **************************************************************************** -->
-  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' ">
+  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' AND $(SonarQubeBuildTasksAssemblyFile) == '' ">
     <!-- Assume that the tasks assembly is in the same location as this targets file 
          or in a parent directory unless another location has already been specified. -->
-    <SonarQubeBuildTasksAssemblyFile Condition=" $(SonarQubeBuildTasksAssemblyFile) == '' ">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), SonarQube.MSBuild.Tasks.dll))\SonarQube.MSBuild.Tasks.dll</SonarQubeBuildTasksAssemblyFile>
+    <SonarQubeBuildTasksAssemblyFile>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), SonarQube.MSBuild.Tasks.dll))SonarQube.MSBuild.Tasks.dll</SonarQubeBuildTasksAssemblyFile>
```

Comment:
I noticed when testing that the we were getting a double backslash because the string returned by GetDirectoryNameOfFileAbove ends with a backslash i.e. I think the example on the MSDN page has a redundant backslash.